activerecord/lib/active_record/associations.rb

```diff
@@ -1649,6 +1656,7 @@ def has_one(name, scope = nil, **options)
         #   belongs_to :comment, touch: true
         #   belongs_to :company, touch: :employees_last_updated_at
         #   belongs_to :user, optional: true
+        #   belongs_to :user, message: 'must be present.'
```

Comment:
What do you think about showing this example with both `:required` and `:message` (`:validation_message`) set, since those two goes together?

Reply:
👍